interfaces/odata.js

```diff
@@ -19,8 +19,8 @@ declare module 'odata' {
   };
 
   declare type OHandler<TModel> = {
-     inlineCount: number,
```

Comment:
I too, but this is how it is inside o.js.. it has `inlineCount()` to add counting to request, and `inlinecount` as result..we probably need to change it inside our o.js fork to `count`  for result